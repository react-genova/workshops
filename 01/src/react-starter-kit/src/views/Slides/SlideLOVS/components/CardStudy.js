//@flow
import React from 'react';
import { Card, CardElement } from '../../../Card';

const CardStudy = () => (
    <Card title="In-depth technological study" description="not enough? some more stuff">
        <CardElement title="project folder structure" description="really, are you serious?" url=""  />
        <CardElement title="middleware in redux" description="the redux source of power" url=""  />
        <CardElement title="Higher Order Components" description="how inheritance works in math" url=""  />
        <CardElement title="avoiding side effects" description="redux-saga, redux-effects, redux-cycles" url=""  />
        <CardElement title="npm and package.json" description="I am root" url=""  />
        <CardElement title="Functional Reactive Programming" description="FRP or F&RP, the long fight" url=""  />
    </Card>
);

export default CardStudy;