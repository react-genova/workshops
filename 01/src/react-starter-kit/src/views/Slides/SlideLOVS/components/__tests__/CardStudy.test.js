//@flow
import React from 'react';
import { shallow } from 'enzyme';
import CardStudy from '../CardStudy';

describe('Card Study', () => {

    it('snapshot check', () => {
        const component = shallow(<CardStudy />);
        expect(component).toMatchSnapshot();
    });

});