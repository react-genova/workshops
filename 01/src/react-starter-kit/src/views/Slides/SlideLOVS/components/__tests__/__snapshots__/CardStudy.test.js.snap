// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Study snapshot check 1`] = `
<Card
  description="not enough? some more stuff"
  title="In-depth technological study"
>
  <CardElement
    description="really, are you serious?"
    title="project folder structure"
    url=""
  />
  <CardElement
    description="the redux source of power"
    title="middleware in redux"
    url=""
  />
  <CardElement
    description="how inheritance works in math"
    title="Higher Order Components"
    url=""
  />
  <CardElement
    description="redux-saga, redux-effects, redux-cycles"
    title="avoiding side effects"
    url=""
  />
  <CardElement
    description="I am root"
    title="npm and package.json"
    url=""
  />
  <CardElement
    description="FRP or F&RP, the long fight"
    title="Functional Reactive Programming"
    url=""
  />
</Card>
`;
